app/src/main/res/values/strings.xml

```diff
@@ -151,16 +151,16 @@
     <string name="delete">Delete</string>
     <string name="save_as_profile">Save as profile</string>
     <string name="save_profile_enter_name">Enter the name of the filter profile:</string>
-    <string name="filterprofiles_empty_state">You have not yet saved any filter profiles.</string>
-    <string name="welcome_to_evmap">Welcome to EVMap</string>
+    <string name="filterprofiles_empty_state">You have no filter profiles saved.</string>
+    <string name="welcome_to_evmap">Welcome</string>
     <string name="welcome_1">Find electric vehicle chargers around you.</string>
     <string name="welcome_2_title">You\'ve got the power</string>
-    <string name="welcome_2">The color of a charger on the map shows you its maximum charging power.</string>
-    <string name="welcome_2_detail">(You can check the colors again under “About EVMap → FAQ” in the menu)</string>
-    <string name="donation_dialog_title">Thank you for using EVMap!</string>
-    <string name="donation_dialog_detail">EVMap is free and Open Source software that I develop in my spare time. Coding contributions on GitHub are very much appreciated. However, due to increasing popularity of the app, I also need to cover some running costs, e.g. for access to the data sources. Therefore, please consider supporting the app through a donation or via GitHub Sponsors.</string>
+    <string name="welcome_2">Chargers are colored based on max charging power.</string>
+    <string name="welcome_2_detail">(Also shown in “About” → “FAQ”)</string>
+    <string name="donation_dialog_title">Thank you for using EVMap.</string>
+    <string name="donation_dialog_detail">Libre software. Its popularity and code contributions are much appreciated. Do help cover the data costs. Pay what it is worth to you.</string>
     <string name="chargeprice_donation_dialog_title">You\'re a real bargain hunter!</string>
-    <string name="chargeprice_donation_dialog_detail">It seems like you like the price comparison feature a lot. To access the pricing data, the developer of EVMap needs to pay a monthly fee to the data provider Chargeprice.app. Therefore, please consider supporting EVMap through a donation.</string>
+    <string name="chargeprice_donation_dialog_detail">You make great use of the price comparison. You can support EVMap through a donation to pay for the data it uses.</string>
```

Comment:
maybe *to pay for the data it uses* -> *to help pay for access to the pricing data*?

Reply:
Access is already there, use of it is what costs something(?).